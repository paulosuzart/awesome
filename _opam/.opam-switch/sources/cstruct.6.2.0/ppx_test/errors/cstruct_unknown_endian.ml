[%%cstruct
  type t =
  { a : uint8_t
  }
[@@unknown_endian]
]
