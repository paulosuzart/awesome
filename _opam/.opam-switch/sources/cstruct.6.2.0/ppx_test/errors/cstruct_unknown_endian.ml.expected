Error: ppx_cstruct: unknown endian unknown_endian, should be little_endian, big_endian, host_endian or bi_endian
